src/Controller/BlogController.php

```diff
@@ -46,9 +47,10 @@ class BlogController extends AbstractController
      * Content-Type header for the response.
      * See https://symfony.com/doc/current/quick_tour/the_controller.html#using-formats
      */
-    public function index(int $page, string $_format, PostRepository $posts): Response
+    public function index(Request $request, int $page, string $_format, PostRepository $posts, TagRepository $tags): Response
     {
-        $latestPosts = $posts->findLatest($page);
+        $tag = $tags->findOneBy(['name' => $request->query->get('tag')]);
```

Comment:
I think this should be wrapped in a check whether the query string param is used, to avoid doing a DB query to load a tag when you don't have a tag name.